### PR TITLE
feat: Add support for bunny hop

### PR DESCRIPTION
**Pull request by mzhu25**
# Description

# Testing Instructions

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.

### CODE REVIEW

**Comment by mzhu25** [comment]
deploy staging

File: src/utils/service_utils.ts [hunk 1]
```diff
@@ -171,19 +171,22 @@ export const serviceUtils = {
...
-                    ...acc,
-                    {
-                        name: source === ERC20BridgeSource.Native ? '0x' : source,
-                        proportion: new BigNumber(percentage.toPrecision(PERCENTAGE_SIG_DIGITS)),
-                    },
-                ];
-            },
-            breakdown,
-        );
+        return Object.entries({ ...defaultSourceBreakdown, ...sourceBreakdown }).reduce((acc, [source, breakdown]) => {
```
Comment (dekz) [inline src/utils/service_utils.ts:174]: still need to rename `Native` to `0x` for consumers

**Comment by dekz** [comment]
deploy staging

**Comment by dekz** [comment]
deploy production

**Pull request merged**